lib/vsc/mympingpong/pairs.py

```diff
@@ -271,13 +270,13 @@ class Shift(Pair):
 
     def new(self, rngarray, iteration):
         # shift through rngarray and convert to a matrix with height = len(self.rng)/2 and width = 2
-        b = n.roll(rngarray, self.offset+iteration).reshape(len(self.rng)/2, 2)
+        b = n.roll(rngarray, self.offset + iteration).reshape(len(self.rng) / 2, 2)
 
         try:
             # n.where(b == self.pairid)[0] returns a list of indices of the elements in b that equal pairid
             # b[n.where(b == self.pairid)[0][0]] is the first element of b that equals pairid
             res = b[n.where(b == self.pairid)[0][0]]
-        except Exception as err:
+        except IndexError as _:
```

Comment:
if not, it will get raised and people can open an issue (i mean, i have no clue, but reading the comment i'd say it's only one we could reasonably expect, and should handle in `try/except`)